ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-master.yaml

```diff
@@ -83,6 +83,22 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws
+- as: e2e-aws-ovn-techpreview
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+    test:
+    - as: e2e-pao-mixed-cpus
+      cli: latest
+      commands: CLUSTER=mcp-only make cluster-label-worker-cnf cluster-deploy-pao
+        pao-functests-mixedcpus
```

Comment:
Why in a new line? Commands for all the other jobs are in a single line, should we do the same here?

Reply:
The call to `make ci-operator-config`  added the new automatically.
We probably crossed the configured threshold here.

Reply:
Ack, Thanks for clarifying.